src/main/resources/assets/merequester/lang/pt_br.json

```diff
@@ -0,0 +1,31 @@
+{
+    "itemGroup.merequester": "ME Solicitante",
+  
+    "item.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "block.merequester.requester": "ME Solicitante",
+  
+    "gui.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "gui.merequester.no_requesters": "Nenhum solicitante de ME encontrado!",

```

Comment:
Have you made sure this fits on the screen? It's the text that is displayed inside the terminal when no ME Requesters are found on the network.